onnxruntime/python/tools/quantization/__init__.py

```diff
@@ -2,3 +2,5 @@
 from .quantize import QuantizationMode
 from .calibrate import CalibrationDataReader, CalibraterBase, MinMaxCalibrater, create_calibrator, CalibrationMethod
 from .quant_utils import QuantType, QuantFormat, write_calibration_table
+from .registry import QLinearOpsRegistry
+from .qdq_quantizer import QDQQuantizer
```

Comment:
why do you expose those 2 APIs?

Reply:
It would better to keep the API simple: just call the quantize_static

Reply:
Removed these APIs.